backend/test/recommendation.js

```diff
@@ -43,6 +43,8 @@ describe('recommendation API', () => {
                         placements[i].should.have.property('description_role');
                         placements[i].should.have.property('employer_name');
                         placements[i].should.have.property('skills');
+                        placements[i].should.have.property('statuts');
```

Comment:
this is a typo.. did you run the tests? this one should have failed

Reply:
ok, I fixed this. I ran the test and it worked, so probably something stupid happened after that, but honestly I don't remember.